packages/bls12-381/bls12-381.0.3.15/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "OCaml binding for bls12-381 from librustzcash"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "conf-rust" {build}
+  "ocaml" {>= "4.07"}
```

Comment:
```suggestion
  "ocaml" {>= "4.07" | (>= "4.08" & with-test)}
```

Reply:
Tests are using the Option module which was introduced in 4.08

Reply:
```suggestion
  "ocaml" {>= "4.07"}
  "ocaml" {with-test & >= "4.08"}
```
The version using the disjunction will not work in this case as:
* `with-test = true & ocaml = 4.07 ---> {>= "4.07" | (>= "4.08" & true)} ---> {true | false} ---> true`

Reply:
Indeed, well spotted!

Reply:
I will restrict to >= 4.08, it is simpler and cleaner.